Lang/Spanish.pj.Lang

```diff
@@ -419,7 +420,7 @@
 //Edit Cheat
 #1027#  "Editar Truco"
 #1028#  "Actualizar Truco"
-#1029#  "El Truco ha sido cambiado.\n\n¿Deseas actualizar?"
+#1029#  "El Truco ha sido cambiado.\n\n¿Quieres actualizar?"

```

Comment:
Oh, and I don't ask you to take my word for it. Check the RAE definition ;)

I'll provide more convincing examples later.